python/MetrologyData.py

```diff
@@ -202,6 +202,22 @@ def resids_boxplot(self, yrange=None, title=None):
         win.set_title(title)
         return win
 
+    # Below applies outlier clipping as in plot_statistics
+    def resids_clip_boxplot(self, nsigma=4, yrange=None, title=None):
+        win = plot.Window()
+	dz = self.resids
```

Comment:
I'm remembering now that I wrote it.  The code did not look familiar, but I pulled it together some time ago from resids_boxplot and plot_statistics to confirm for myself that Peter's equivalent plots were made after he removed outliers.